go/database/mpt/tool/export_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package main
+
+import (
+	"context"
+	"syscall"
+	"testing"
+	"time"
+)
+
+func Test_CatchInterrupt(t *testing.T) {
```

Comment:
I think this is too trivial test - it mostly tests only golang build-in function.

You should test that really export can be interrupted. It may be tricky timing wise, but perhaps it could be done via a mocked io.Writer, which will be very slow. It will take e.g a second to accept data, then you could have enough time to  trigger cancel even when exporting a small sample of data